docs/cog_guides/core.rst

```diff
@@ -0,0 +1,3393 @@
+
+""""""""""""
+set username
+""""""""""""
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]set username <username>
+
+.. tip:: Alias: ``set name``
+
+**Description**
+
+Sets Red's username.
+
+Maximum length for a username is 32 characters.
+
+.. Note:: The username of a verified bot cannot be manually changed.
+
+    Please contact Discord support to change it.
+
+Example:
+    - ``[p]set username BaguetteBot``
+
+**Arguments:**
+
+- ``<username>`` The username to give the bot.
+
+.. _core-command-set-watching:
+
+""""""""""""
+set watching
+""""""""""""
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]set watching [watching]
+
+**Description**
+
+Sets Red's watching status.
+
+This will appear as ``Watching <watching>``.
+
+Maximum length for a watching status is 128 characters.
+
+Examples:
+    - ``[p]set watching`` - Clears the activity status.
+    - ``[p]set watching [p]help``
+
+**Arguments:**
+
+- ``[watching]`` The text to follow ``Watching``. Leave blank to clear the current activity status.
+
+.. _core-command-shutdown:
+
+^^^^^^^^
+shutdown
+^^^^^^^^
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]shutdown [silently=False]
+
+**Description**
+
+Shuts down the bot.
+
+Allows Red to shut down gracefully.
+
+This is the recommended method for shutting down the bot.
+
+.. _core-command-traceback:
+
+^^^^^^^^^
+traceback
+^^^^^^^^^
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]traceback [public=False]
+
+**Description**
+
+Sends to the owner the last command exception that has occurred.
+
+If public (yes is specified), it will be sent to the chat instead.
+
+.. Warning:: Sending the traceback publicly can accidentally reveal sensitive information about your computer or configuration.
+
+
+Examples:
+    - ``[p]traceback`` - Sends the traceback to your DMs.
+    - ``[p]traceback True`` - Sends the last traceback in the current context.
+
+**Arguments:**
+
+- ``[public]`` Whether to send traceback to the current context. Leave blank to send to DMs.
+
+.. _core-command-unignore:
+
+^^^^^^^^
+unignore
+^^^^^^^^
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]unignore 
+
+**Description**
+
+Commands to remove servers or channels from the ignore list.
+
+.. _core-command-unignore-channel:
+
+""""""""""""""""
+unignore channel
+""""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]unignore channel [channel]
+
+**Description**
+
+Remove a channel or category from the ignore list.
+
+Defaults to the current channel.
+
+Examples:
+    - ``[p]unignore channel #general`` - Unignores commands in the #general channel.
+    - ``[p]unignore channel`` - Unignores commands in the current channel.
+    - ``[p]ignore channel "General Channels"`` - Use quotes for categories with spaces.
+    - ``[p]unignore channel 356236713347252226`` - Also accepts IDs. Use this method to ignore categories.
```

Comment:
```suggestion
    - ``[p]unignore channel 356236713347252226`` - Also accepts IDs. Use this method to unignore categories.
```

Reply:
Accepted and an additional ignore needed to be unignore on 3321